meerk40t/core/elements.py

```diff
@@ -207,6 +285,10 @@ def __init__(self, kernel, index=None, *args, **kwargs):
 
         self.setting(bool, "classify_reverse", False)
         self.setting(bool, "legacy_classification", False)
+        self.setting(bool, "classify_fuzzy", False)
+        self.setting(float, "classify_fuzzydistance", 100.0)
+        self.setting(bool, "classify_autogenerate", True)
+        self.setting(bool, "classify_default", True)
```

Comment:
These are actually redundant. If the choice above is registered against a context it'll automatically set those values.

Reply:
To be honest I wasn't aware of this, a code tinkerer like myself might look for clearer evidence of a new context variable being established. So as it is not hurting either I vote for maintaining this bit of code, but as in everything else: your project, your call